src/common/obj_bencher.cc

```diff
@@ -17,6 +17,7 @@
  */
 #include <pthread.h>
 #include "common/Cond.h"
+#include "include/compat.h"
```

Comment:
I'd be interested to hear of those problems since I'm currently recommending to put it as first include.
Like I actually witnessed: It prevents redefinition of some values.
And the problem with ENODATA <> ENOATRR is real and so easy to fix otherwise.